src/App.jsx

```diff
@@ -14,70 +15,76 @@ export const goods = [
+    setSelectedGood('');
+  }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  function addGood(goodItem) {
+    setSelectedGood(goodItem);
+  }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      {selectedGood ? (
+        <h1 className="title is-flex is-align-items-center">
+          {`${selectedGood} is selected`}
+          <button
+            onClick={onClearButton}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(goodItem => (
+            <tr
+              key={goodItem}
+              data-cy="Good"
+              className={
+                selectedGood === goodItem
+                  ? 'has-background-success-light'
+                  : ''
+              }
             >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+              <td>
+                {selectedGood !== goodItem && (
+                  <button
+                    onClick={() => addGood(goodItem)}
+                    data-cy="AddButton"
+                    type="button"
+                    className="button is-normal is-success"
+                  >
+                    +
+                  </button>
+                )}
+                {selectedGood === goodItem && (
+                  <button
+                    onClick={onClearButton}
+                    data-cy="RemoveButton"
+                    type="button"
+                    className="button is-info"
+                  >
+                    -
+                  </button>
+                )}
```

Comment:
```suggestion
                {selectedGood === goodItem ? (
                  <button
                    onClick={onClearButton}
                    data-cy="RemoveButton"
                    type="button"
                    className="button is-info"
                  >
                    -
                  </button>
                ) : (
                  <button
                    onClick={() => addGood(goodItem)}
                    data-cy="AddButton"
                    type="button"
                    className="button is-normal is-success"
                  >
                    +
                  </button>
                )}
```
One condition and a ternary operator will do the job, you don't need two separate conditions with an `&&` operator.
Also, if you structure your code in a different way, you can remove duplication of the `<button>` tag.